### PR TITLE
remove NOGLYPHS (only)

### LINKED ISSUE #1129: Petaluma Font: Remove NOGLYPH glyphs
238 of the glyphs in `petaluma_glyphs.ts` have an outline that renders a [NOGLYPH] image.

Search for
```
m 1181 1440 l 268 1440 l 268 -1440 l 1181 -1440 z m 959 -524 l 959 -850 l 481 -850 l 481 -524 z m 864 -619 l 576 -619 l 576 -755 l 864 -755 z m 959 -26 l 959 -351 l 481 -351 l 481 -26 l 576 -26 l 576 -256 l 864 -256 l 864 -121 l 769 -121 l 769 -187 l 671 -187 l 671 -26 z m 959 325 l 959 37 l 481 37 l 481 135 l 864 135 l 864 325 z m 959 498 l 959 403 l 769 403 l 769 202 l 481 202 l 481 297 l 671 297 l 671 403 l 481 403 l 481 498 z m 671 778 l 576 778 l 576 674 l 671 674 z m 769 876 l 769 674 l 959 674 l 959 576 l 481 576 l 481 876 z m 959 1247 l 959 1152 l 769 1152 l 769 1045 l 959 1045 l 959 950 l 481 950 l 481 1045 l 671 1045 l 671 1152 l 484 1152 l 484 1247 z m 959 -916 l 959 -1011 l 867 -1011 l 665 -1146 l 959 -1146 l 959 -1241 l 481 -1241 l 481 -1146 l 683 -1011 l 481 -1011 l 481 -916 z
```

We should remove these glyphs, and then update the FONT_STACKS in vexflow_test_helpers to use Bravura as a fallback font for Petaluma. That way, when a glyph is missing in Petaluma, it just pulls the glyph from Bravura (which should include most of the glyphs we care about).

### LINKED ISSUE #1188: custom font glyphs (custom.js)
This issue is just a reminder to myself to figure out how to handle custom.js going forward. Are there glyphs we should remove from custom.js (in case Bravura includes them already?). Which projects are using these glyphs?

---

The first four glyphs defined in custom.js are also present (and identical) in gonville.js.
```
v39: {
  x_min: -171.5,
  x_max: 251.8125,
  ha: 257,
  o: '...',
},
v7a: {
  x_min: -1.359375,
  x_max: 386.5625,
  ha: 394,
  o: '...',
},
v8d: {
  x_min: -1.359375,
  x_max: 255.890625,
  ha: 261,
  o: '...',
},
vb7: {
  x_min: -176.9375,
  x_max: 251.8125,
  ha: 257,
  o: '...',
},
```

Looking into valid_codes.js, we see that:
* v39 == accidentalBuyukMucennebFlat
* v7a == vexAccidentalMicrotonal2
* v8d == accidentalKucukMucennebSharp
* vb7 == accidentalBakiyeFlat

I don't know why just these four are duplicated between custom and gonville, but we can investigate at some point.

The rest of the custom glyphs are not duplicated anywhere else. The mappings are:
* vd0 == accidentalSori
* vd1 == accidentalKoron
* vd2 == noteheadSquareBlack
* vd3 == noteheadSquareWhite
* vd4 == vexNoteHeadRectBlack
* vd5 == vexNoteHeadRectWhite
* vd6 == vexAccidentalMicrotonal3
* vd7 == vexAccidentalMicrotonal4

```
vd0: {
  x_min: -10.890625,
  x_max: 299.4375,
  ha: 294,
  o:
},
vd1: {
  x_min: -20,
  x_max: 320,
  ha: 257,
  o:
},
vd2: {
  x_min: 0,
  x_max: 430.75,
  ha: 386, 
  o:
},
vd3: {
  x_min: 0,
  x_max: 430.75,
  ha: 386,
  o:
},
vd4: {
  x_min: 0,
  x_max: 430.75,
  ha: 360, 
  o:
},
vd5: {
  x_min: 0,
  x_max: 430.75,
  ha: 360,
  o:
},
vd6: {
  x_min: 0,
  x_max: 430.75,
  ha: 360,
  o:
},
vd7: {
  x_min: 0,
  x_max: 430.75,
  ha: 360,
  o:
},
```

### PR DESCRIPTION
**Pull request by rvilarl**
@0xfe as wished I have extracted the removal of NOGLYPHS from #1242.

Fixes #1129
- Fonts regenarated from OTF in order to remove the NOGLYPH which were most probably added manually.
- Valid code enhanced to include Brass Techniques and symbol required by Additive/Complex Signatures. 

Fixes #1188

### CODE REVIEW

File: src/fonts/bravura_glyphs.ts [hunk 1]
```diff
@@ -331,130 +371,130 @@ export const BravuraFont = {
...
+      o: 'm 452 217 b 384 194 426 217 408 207 b 350 176 374 189 363 176 b 343 177 347 176 344 176 b 328 202 333 180 328 190 b 337 606 328 232 337 579 b 310 629 337 622 324 629 b 265 592 291 629 268 616 b 269 194 265 592 266 395 b 196 217 248 209 223 217 l 187 217 b 120 194 160 217 143 207 b 85 176 108 189 98 176 b 78 177 82 176 81 176 b 62 202 68 180 62 190 b 72 606 62 232 72 579 b 45 629 72 622 59 629 b 0 592 24 629 1 616 b 17 -245 0 592 7 -232 b 30 -252 22 -251 26 -252 b 39 -249 35 -252 39 -249 b 153 -161 84 -226 120 -186 b 274 -50 206 -124 246 -85 b 284 -245 276 -158 279 -240 b 295 -252 288 -251 292 -252 b 304 -249 300 -252 304 -249 b 416 -163 348 -226 383 -186 b 592 82 546 -72 592 16 b 461 217 592 163 528 216 z m 96 -134 b 84 -138 92 -137 86 -138 b 68 -115 72 -138 68 -124 b 63 26 68 -115 63 -30 b 66 72 63 49 65 66 b 168 144 75 101 134 144 b 226 59 212 144 226 95 b 96 -134 226 -22 161 -94 z m 361 -134 b 348 -138 356 -137 351 -138 b 333 -115 337 -138 333 -124 b 328 39 333 -115 328 -16 b 330 72 328 55 328 68 b 433 144 338 101 399 144 b 468 130 448 144 459 140 b 490 59 481 112 490 88 b 361 -134 490 -22 426 -94 z',
```
Comment (0xfe) [inline src/fonts/bravura_glyphs.ts:489]: Do you know why all the glyphs ended up changing here?

**Review by 0xfe — COMMENTED** [review]
Any visual diffs here?

File: src/fonts/bravura_glyphs.ts (hunk 1 above)
Comment (rvilarl) [reply src/fonts/bravura_glyphs.ts:489]: My theory is that `bravura_glyphs.ts` was generated with an older version of Bravura and not with version 1.392 which is the one in `tools/fonts/@/bravura`

Comment (rvilarl) [reply src/fonts/bravura_glyphs.ts:489]: The values that I have commited were automatically generated using `node fontgen_smufl.js @/bravura/Bravura_1.392.otf bravura_glyphs.ts`

Comment (rvilarl) [reply src/fonts/bravura_glyphs.ts:489]: As I commented in #1242:

> one or two instance with changes that cannot be appreciated

I will post the changes below.

**Comment by rvilarl** [comment]
**Annotation Bottom_Annotations_with_Beams Bravura**
Annotation Bottom_Annotations_with_Beams Bravura
**Current**
Annotation Bottom_Annotations_with_Beams Bravura_Current
**Master**
Annotation Bottom_Annotations_with_Beams Bravura_Reference

**Comment by rvilarl** [comment]
**Dot Basic Bravura**
Dot Basic Bravura
**Current**
Dot Basic Bravura_Current
**Master**
Dot Basic Bravura_Reference

**Comment by rvilarl** [comment]
**Ornament Jazz_Ornaments Bravura**
Ornament Jazz_Ornaments Bravura
**Current**
Ornament Jazz_Ornaments Bravura_Current
**Master**
Ornament Jazz_Ornaments Bravura_Reference

**Comment by rvilarl** [comment]
No other visual differences apart from the ones related with NOGLYPHS.

**Comment by 0xfe** [comment]
Great, thanks Rodrigo!

**Pull request merged**